package.json

```diff
@@ -29,17 +29,17 @@
     "fuzzysort": "^1.1.4",
     "parcel": "^1.12.4",
     "prettier": "^2.0.5",
-    "tailwindcss": "^1.4.6",
+    "tailwindcss": "^2.0.3",
     "to-string-loader": "^1.1.6",
     "vuepress": "^1.2.0"
   },
   "dependencies": {
     "lodash.max": "^4.0.1",
     "lodash.range": "^3.2.0",
-    "tailwindcss-typography": "^3.0.0"
+    "tailwindcss-typography": ">=3.0.0"
   },
   "peerDependencies": {
-    "tailwindcss": ">=1.4.6"
+    "tailwindcss": ">=2.0.3"
```

Comment:
We need to keep the v1 constraint along the v2 to be backward compatible and avoid a breaking change. 

```json
  "tailwindcss": ">=1.4.6 || >=2.0.3"
```